Public/Src/Utilities/Native/IO/Windows/FileSystem.Win.cs

```diff
@@ -2900,7 +2919,18 @@ public bool TryGetFileAttributes(string path, out FileAttributes attributes, out
                 }
                 else
                 {
-                    return new NativeFailure(hr);
+                    // Fall back using more expensive FindFirstFile.

```

Comment:
>back [](start = 28, length = 4)

Perhaps copy part of the comment from your PR here on why we have to handle this NTFS quirk